### DISCUSSION TITLE
node项目怎么调用libpag依赖中的方法

### DISCUSSION THREAD

**Discussion by guileryu** [Tencent/libpag#2102] [Q&A] [answered]
![C70F4D89-BE45-41dc-9622-8C2005735D2B](https://github.com/Tencent/libpag/assets/59436015/886fd927-b8be-41cd-bbe8-779849859c01)

node test.mjs 执行完一直报TypeError: Failed to parse URL from libpag.wasm
有没有node服务端执行相关的文档

**Comment by guileryu** [comment]
@domchen 这个帮我看下，有没nodejs的对接文档

**Reply by domchen** [reply, marked as answer]
不支持node，需要浏览器环境。
